src/lib/context-menu.ts

```diff
@@ -256,6 +261,35 @@ export class ContextMenu {
         })
       )
     }
+    if (type === schema.nodes.table_element) {
```

Comment:
This is a duplicated check. Please add your piece of code under the existing `if (type === schema.nodes.table_element)` condition in line 294.